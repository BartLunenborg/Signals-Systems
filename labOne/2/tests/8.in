100.0 -1.57
