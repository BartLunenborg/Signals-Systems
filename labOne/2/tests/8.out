0.08 -100.00
